bin/anthology/papers.py

```diff
@@ -181,7 +181,8 @@ def from_xml(xml_element, *args):
                 [x[0].full for x in paper.attrib["author"]]
             )
 
-        paper.attrib["citation"] = paper.as_markdown()
```

Comment:
Why did you comment out this line?

Reply:
It was causing errors and I was trying to get it done quickly. Accidental commit. It turned out it was caused by an empty `<venue>` tag.